lib/logic.rb

```diff
@@ -0,0 +1,79 @@
+class TicTacToe
+  attr_accessor :player
+  attr_reader :board, :game, :player_char, :loc, :move, :state, :token
```

Comment:
- [ ] Remove unnecessary attr_reead/accessor values that you won't use anywhere else.